tealer/teal/instructions/instructions.py

```diff
@@ -3903,6 +4847,14 @@ def start_position(self) -> int:
             raise TealerException("replace instruction does not have any immediates")
         return self._idx
 
+    @property
```

Comment:
Number of values popped depends on the immediate argument `s`. If there's **no** immediate argument then, `replace` is `replace3` and `stack_pop_size = 3`. If there is a immediate argument then `replace s` is equivalent to `replace2 s` and `stack_pop_size = 2`.